### ISSUE TITLE
Error when switching to Windows containers

### ISSUE THREAD

**Issue by jtnord** [docker/for-win#304]
### Expected behaviour

When I "switch to windows containers" in the docker app it should work.

### Actual behavior

It failed with:

```
Docker daemon failed with message:
time="2016-12-09T13:48:00.645679100Z" level=info msg="Windows default isolation mode: hyperv" 
time="2016-12-09T13:48:00.766311500Z" level=info msg="Graph migration to content-addressability took 0.00 seconds" 
time="2016-12-09T13:48:00.766311500Z" level=info msg="Loading containers: start." 
Error starting daemon: Error initializing network controller: Error creating default network: HNS failed with error : The parameter is incorrect. 


   at Docker.Backend.ContainerEngine.Windows.DoStart(Settings settings)
   at Docker.Backend.ContainerEngine.Windows.Start(Settings settings)
   at Docker.Core.Pipe.NamedPipeServer.<>c__DisplayClass8_0.<Register>b__0(Object[] parameters)
   at Docker.Core.Pipe.NamedPipeServer.RunAction(String action, Object[] parameters)
```
### Information

  - Diagnostic ID 4C36067F-0848-4ECE-B0F4-887DD80EA81B/2016-12-09_13-55-28
  - a reproducible case if this is a bug.
  - page URL if this is a docs issue or the name of a man page: N/a
  - host distribution and version (Windows version, build number, etc)

```
Version: 1.13.0-rc3-beta32 (9266)
Channel: Beta
Sha1: 3990397e132a10772955c122b0ca9258fdaf84ce
Started on: 2016/12/09 13:41:21.726
Resources: C:\Program Files\Docker\Docker\Resources
OS: Windows 10 Pro
Edition: Professional
Id: 1607
Build: 14393
```

a crash report has been uploaded with id `4C36067F-0848-4ECE-B0F4-887DD80EA81B/2016-12-09_13-48-01`

### Steps to reproduce the behavior

  1. Run Switch to windows containers
  2. boom

**Comment by rneugeba** [comment]
Could you try running the debug script from Microsoft found [here](https://github.com/Microsoft/Virtualization-Documentation/tree/live/windows-server-container-tools/Debug-ContainerHost). It diagnoses common issues with Windows containers.

For networking, there is also @JMesser81  PR for additional checks [here](https://github.com/Microsoft/Virtualization-Documentation/pull/513)

**Comment by jornh** [comment]
After the update to `beta32.1` from `beta31` AFAIK I got bitten by this too with `Error creating default network: HNS failed with error : The parameter is incorrect.`

https://github.com/docker/for-win/issues/90#issuecomment-254678067 helped me with a fix as it turned out `Get-ContainerNetwork` didn't return a NAT any more. If this was due to the the Docker update or something else is hard for me to prove now. 😄 Anyways `New-ContainerNetwork -Name nat -Mode NAT –subnetprefix 10.0.76.0/24` and switching to Windows containers works for me again.
